python/mujincontrollerclient/controllerclientbase.py

```diff
@@ -188,10 +188,10 @@ def Ping(self, usewebapi=True, timeout=5):
         if response.status_code != 200:
             raise ControllerClientError(_('failed to ping controller, status code is: %d') % response.status_code)
 
-    def SetLogLevel(self, level, timeout=5):
+    def SetLogLevel(self, componentLevels, timeout=5):
         """ Set webstack log level
         """
-        response = self._webclient.Request('POST', '/loglevel/', data={'level': level}, timeout=timeout)
+        response = self._webclient.Request('POST', '/loglevel/', data={'componentLevels':componentLevels}, timeout=timeout)
```

Comment:
space ; 0

Reply:
All right, I also added JSON serialization so that we can configure multiple loggers.